src/Components/BoxColor/BoxColor.js

```diff
@@ -0,0 +1,18 @@
+import "./BoxColor.css"
+
+const BoxColor = (props) => {
```

Comment:
En general, de cara a todos los componentes puedes descomponer las props en el input de la función.

```suggestion
const BoxColor = ({r,g,b}) => {
```